frontend/cypress/support/auth.ts

```diff
@@ -5,16 +5,14 @@ const SERVER_URL = Cypress.env('AUTH_URL') || 'http://localhost:8080'
 const ISSUER = 'common'
 const CACHE_ENTRY = 'FUSION_AUTH_CACHE'
 
-/* TODO: figure out if we want to retrieve token before each test,
- * or if we are able to store tokens of all users we need (or all users at all)
- */
+/* Gets the access token from local cache which was stored on login. */
 export const getToken = (): string => {
     const fusionStorageJson = localStorage.getItem(CACHE_ENTRY)
     if (fusionStorageJson === null) {
         throw new Error('Could not find auth token in local storage')
     }
     const fusionStorage = JSON.parse(fusionStorageJson)
-    const token = fusionStorage[`${CACHE_ENTRY}:8829d4ca-93e8-499a-8ce1-bc0ef4840176:TOKEN`]
```

Comment:
What is the GUID here?

Reply:
It's the client ID. I'll change this to match what we are doing in `frontend/src`